main.py

```diff
@@ -1,10 +1,22 @@
+import pymongo
 from typing import Union
 
-from fastapi import FastAPI
+from pymongo import MongoClient
+from fastapi import FastAPI, Request
 
 app = FastAPI()
 
 
-@app.get("/")
+@app.on_event("startup")
+def startup():
+    print("startup")
+
+    client = MongoClient('mongodb://localhost:8081/')
+    app.state.client = client
+    app.state.db = client['inflation']
+    app.state.products = app.state.db['products']
```

Comment:
All these settings 

* `mongodb://localhost:8081/`
* `inflation`
* `products`

should be placed in `settings.py` module (in the project root). For example,

```python
MONGO_URI = 'mongodb://localhost:8081/'
```

Also, all sensitive information must be passed to the app through environment variables. See [environs](https://pypi.org/project/environs/) package